app/models/solidus_subscriptions/installment.rb

```diff
@@ -23,6 +23,10 @@ class Installment < ApplicationRecord
       where.not(id: fulfilled_ids).distinct
     end)
 
+    scope :active, (lambda do
```

Comment:
I don't know if `Installment.active` is very meaningful: installments by themselves cannot be active or inactive. Maybe it's best to go with a longer name such as `with_active_subscription`?

Reply:
Yeah, that makes sense. Thanks